Python/perf_trampoline.c

```diff
@@ -484,11 +494,11 @@ _PyPerfTrampoline_Init(int activate)
         return -1;
     }
     if (!activate) {
-        tstate->interp->eval_frame = NULL;
+        set_eval_frame(tstate, NULL);
```

Comment:
I think that should be fine (the tests pass using it). @pablogsal - do you know if we need to set `tstate->eval_frame` explicitly when setting / clearing the perf trampoline, or is it fine to go through `_PyInterpreterState_SetEvalFrameFunc()`?

Reply:
I don't think there is any difference here so it should be safe to just go through `_PyInterpreterState_SetEvalFrameFunc()`.

Just remember to run the buildbots before landing because these tests only run there with all the options.